### ISSUE TITLE
Book Authorization

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/PaDS#10]
Like a journal issue, some PEP books are split.  Currently, PaDS isn't giving out "true" access permits for a book that does not fall on the complete book ID's initial page.

Example::

When you authorize a journal article, you are looking at the journal code, and volume, which is part 1 and 2 of the PEP article ID, e.g., **_IJP.077_**.0022

Yes, you also asked for year, but in 99% of the cases, volume and year refer to the same data set.  (As an aside: Year information, in fact, is unreliable...Taylor and Francis keep coding Publication year as the year they put the information out as XML, but in fact, as far as journal is concerned, the volume and the year are in sync.  I correct that data when it comes in that way.)

You don't look at the page number to see if the user has permission.

Similarly, for books, I think you need to look only at the first two parts of the ID to authorize.  Those are all that matters.  You might get a permit request for "part of a book", based on page number, but the page numbers don't matter.  

In my software, I refer to the first two parts of the ID as the "basecode", and when I built the original permission system for the server, I often started referring to it as productcode (perhaps more accurately so).  My Productbase table has as it's key, basecode, here are a few partial rows (first two columns) of journals/videos and books:
```
basecode articleid
---------- -----------------
SFCPVS	(Null)
SGS	        (Null)
SPIVS	(Null)
SPR	        (Null)
TVPA	(Null)
UCLVS	(Null)
ZBK002	ZBK.002.0001A - But for some "split" books, this is only the front instance, the "TOC"
ZBK003	ZBK.003.0001A
ZBK004	ZBK.004.0001A
ZBK005	ZBK.005.0001A
ZBK006	ZBK.006.0001A
ZBK007	ZBK.007.0001A
```

In short: I think all you need to do is to grab the first two sections of the PEP article ID I send, and use that as your "product code" to properly authorize books.  It works for journals too, especially if you are authorizing by year.

It should be easy for you to parse out the "basecode" from the articleIID I send in the permits request.  If you use that (basically, just ignore the page number part), the permit system should work for split books as well.

**Comment by jwoosnam** [comment]
@nrshapiro this has now been resolved and released to stage as v06.04i
